libraries/adaptive-expressions/src/builtinFunctions/coalesce.ts

```diff
@@ -24,8 +24,8 @@ export class Coalesce extends ExpressionEvaluator {
         return FunctionUtils.apply((args: any[][]): any => Coalesce.evalCoalesce(args as any[]));
     }
 
-    private static evalCoalesce(objetcList: object[]): any {
-        for (const obj of objetcList) {
+    private static evalCoalesce(objectList: any[]): any {
```

Comment:
I'm leaving avoiding using object as a type fixed for a later PR, except this one which was a private function.